api/src/main/java/org/openmrs/module/mirebalaisreports/definitions/MexicoCesMedsReportManager.java

```diff
@@ -0,0 +1,63 @@
+package org.openmrs.module.mirebalaisreports.definitions;
+
+import org.openmrs.module.mirebalaisreports.MirebalaisReportsProperties;
+import org.openmrs.module.pihcore.config.ConfigDescriptor;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.openmrs.module.mirebalaisreports.definitions.BaseReportManager.Category.DATA_EXPORT;
+
+@Component
+public class MexicoCesMedsReportManager extends BasePihReportManager {
+
+    @Override
+    public Category getCategory() {
+        return DATA_EXPORT;
+    }
+
+    @Override
+    public List<ConfigDescriptor.Country> getCountries() {
+        return Arrays.asList(ConfigDescriptor.Country.MEXICO);
+    }
+
+    @Override
+    public Integer getOrder() {
+        return REPORTING_DATA_EXPORT_REPORTS_ORDER.indexOf(getUuid());
+    }
```

Comment:
Do you need to add MEXICO_CES_MEDS_REPORT_DEFINITION_UUID to this List so that this returns something other than -1?

Reply:
It's there https://github.com/PIH/openmrs-module-mirebalaisreports/pull/11/files#diff-22e11c4f0ad93043cbf259867a310f13R79